cluster-install/install.sh

```diff
@@ -572,7 +573,7 @@ loginfo() {
 }
 
 logwarn() {
-    log_at_level $__WARN "$1"
+    log_at_level $__WARN "WARNING: $1"
```

Comment:
Felt like the warnings kind of flew under the radar a bit too much. Still not sure about adding this though, so def interested to hear what others think about adding this.